pod-operation/src/state_machine.rs

```diff
@@ -187,6 +190,18 @@ impl StateMachine {
 		self.high_voltage_system.disable();
 	}
 
+	fn _enter_faulted(&mut self) {
+		info!("Entering Faulted state");
+		self.io
+			.of("/control-station")
+			.unwrap()
+			.emit("fault", "123")
+			.ok();
```

Comment:
Note: I think we'll eventually want to emit the reason of the fault meaning we'll have to move this to where the fault actually occurs, but we can revisit that as part of #56.

Reply:
perf